toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -238,7 +238,7 @@ make-4.2.1-4.cm1.x86_64.rpm
 make-debuginfo-4.2.1-4.cm1.x86_64.rpm
 mariner-check-macros-1.0-3.cm1.noarch.rpm
 mariner-release-1.0-9.cm1.noarch.rpm
-mariner-repos-1.0-9.cm1.noarch.rpm
+mariner-repos-1.0-10.cm1.noarch.rpm
```

Comment:
Since there is a new package produced by `mariner-repos` (`mariner-repos-preview`) it should be added to the toolchain manifests. The toolchain manifests record all rpms built as part of the toolchain, regardless if they are part of the worker chroot.

Reply:
I do not think we want this here...this is strictly an optional package that can be installed in a running image.  During the build process, I think we simply want to add it as a command line argument when we want to use the preview repo...or something of that nature.

Reply:
Discussed offline, resolving.

---
In reply to: [497070195](https://github.com/microsoft/CBL-Mariner/pull/146#discussion_r497070195) [](ancestors = 497070195)